src/test/java/id/ac/itb/logistik/ditlog/DitlogIndicatorTests.java

```diff
@@ -2,37 +2,42 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import id.ac.itb.logistik.ditlog.model.Indicator;
+import id.ac.itb.logistik.ditlog.model.User;
+import id.ac.itb.logistik.ditlog.service.TokenAuthenticationService;
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.skyscreamer.jsonassert.JSONAssert;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.MediaType;
-import org.springframework.http.ResponseEntity;
+import org.springframework.http.*;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
 public class DitlogIndicatorTests extends BaseTest {
```

Comment:
Since there is now roleId in the payload, the test should also check the value of that